arbnode/inbox_reader.go

```diff
@@ -333,6 +353,7 @@ func (r *InboxReader) run(ctx context.Context, hadError bool) error {
 				return err
 			}
 			if ourLatestDelayedCount < checkingDelayedCount {
+				log.Trace("Expecting to find delayed messages", "checkingDelayedCount", checkingDelayedCount, "ourLatestDelayedCount", ourLatestDelayedCount, "currentHeight", currentHeight)
```

Comment:
all of these should be log.Debug

Trace is a huge log (can ge GBs in less than an hour) that we cannot collect over time.
Debug logs are only for debugging, but you can run a normal node with it and it'll be o.k.